CodeStarterCamp_Week3/main.swift

```diff
@@ -8,5 +8,12 @@
 
 import Foundation
 
-print("Hello, World!")
-
+/* Step 1: 타입과 인스턴스의 관계 */
+var misterLee = Person(name: "misterLee")
+var missKim = Person(name: "missKim", money: 5000)
+let yagombucks = CoffeeShop(menu: [
+    Coffee.americano: 4500,
+    Coffee.latte: 5000,
+    Coffee.cappuccino: 5000,
+    Coffee.macchiato: 5900,
+], barista: misterLee)
```

Comment:
```suggestion
let yagombucks = CoffeeShop(menu: [.americano: 4500,
                                   .latte: 5000,
                                   .cappuccino: 5000,
                                   .macchiato: 5900],
                            barista: misterLee)
```

컨벤션을 위해 개행을 선택하셨군요!

그렇다면 이정도 컨벤션이 어떨까 싶습니다. `,`를 기준으로 개행하는 것이요!

더불어 `menu`의 `key`는 `Coffee` 타입임이 명확하기 때문에 `Coffee`를 생략하고 바로 `.americano`와 같이 적는 것이 더 가독성에 좋을 것 같습니다.